spec/osut_tests_spec.rb

```diff
@@ -1545,5 +1545,13 @@ module M
     expect(mod1.reset(DBG)).to eq(DBG)
     expect(mod1.level).to eq(DBG)
     expect(mod1.clean!).to eq(DBG)
+
+    subs = []
+    subs << {height: 0.2, width: 0.2}
```

Comment:
The most basic (minimal) call to _addSubs_ (all other arguments are defaulted). This generates the tiny, illustrated 8" x 8" side window. All EnergyPlus checks pass. Simulation runs are good. 

<img width="1430" alt="seb_right" src="https://user-images.githubusercontent.com/98104985/236192249-b1ff9aa5-dd0d-481e-9ff3-f72e28956c7b.png">

The method also allows users to request a WWR-calculated sub surface addition, or instead an array of equally-sized sub surfaces. The method can be run on the same _base_ surface over and over, without deleting previously-added sub surfaces, e.g. adding a corner door AFTER adding a WWR-calculated horizontal window band, adding a transom window above a door (or vice versa). A LOT more testing needs to be completed before merging ...